l10n.yaml

```diff
@@ -0,0 +1,3 @@
+arb-dir: lib/l10n
+template-arb-file: app_en.arb
+output-localization-file: app_localizations.dart
```

Comment:
When I run the main I get an error because it looks like that the word dashboard doesn't have a translation in the Italian file, you can check the missing words by adding here `untranslated-messages-file: missingWords.txt`